content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -222,6 +224,70 @@ At this point, the flow is now complete.
    order to confirm the delivery. Otherwise, the delivered quantity can be entered manually directly
    on the sales order.
 
+.. _sales/invoicing/down_payment/100-percent-down-payments:
+
+Example: request 100% down payment
+==================================
+
+The process of requesting a 100% down payment is similar to the process of setting up a :ref:`50%
+down payment <sales/invoicing/down_payment/50-percent-down-payments>` but with fewer steps. The
+*Solar Panel Installation* product is being used in this example.
+
+To configure a 100% down payment, begin by navigating to :menuselection:`Sales app --> New` and add
+a :guilabel:`Customer` to the quote.
+
+Next, click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab, and select the
+:guilabel:`Solar Panel Installation` product.
+
+:guilabel:`Confirm` the order and the quotation will turn into a sales order. An invoice can now be
+created by clicking :guilabel:`Create Invoice` in the top-left.
+
+.. image:: down_payment/100p-order-confirmed.png
+   :align: center
+   :alt: Confirmed sales order for the Solar Panel Installation product.
+
+On the :guilabel:`Create invoices` pop-up window that appears, select :guilabel:`Down payment
+(percentage)`, and type `100` in the :guilabel:`Down Payment Amount` field.
+
+.. image:: down_payment/100p-down-payment-percentage.png
+   :align: center
+   :alt: The Down payment (percentage) option selected with 100% set as the Down Payment.
+
+Next, click :guilabel:`Create Draft Invoice` to create and an invoice draft. This will also
+bring the draft invoice into view, which includes the down payment as a :guilabel:`Product` in the
+:guilabel:`Invoice Lines` tab.
+
+The invoice can now be confirmed and posted by clicking :guilabel:`Confirm`. Confirming the
+invoice changes the status from :guilabel:`Draft` to :guilabel:`Posted`. It also reveals a new
+series of smart buttons at the top of the page.
```

Comment:
During my revisions, using a standard 17.0 runbot, I didn't see a series of smart buttons appear at the top of the page. So, that is worth a quick double-check to ensure the accuracy of that statement. 

UPDATE: I can see on line 268 that you refer to the 'Register Payment' button as a "smart button." That is actually not the case. Totally easy mistake to make, though. The "smart buttons" are the buttons that appear at the top of the page (typically white boxes with figures, icons, etc.) -- the purple and grey buttons that appear on the menu line, slightly above the form are just regular buttons.